src/ui/rcb-ui-inner.svelte

```diff
@@ -36,6 +36,14 @@
   export let purchaseOperationHelper: PurchaseOperationHelper;
   export let isInElement: boolean = false;
 
+  const shouldShowDetailsButton =
+    (Boolean(brandingInfo?.appearance) &&
```

Comment:
We have an explicit flag to show/hide the product description.
So I would check explicitly for that flag.
```suggestion
    (Boolean(brandingInfo?.appearance?.show_product_description) &&
```

Reply:
wow nice catch, I did have it like that and totally removed it without realizing

Reply:
done